CMakeLists.txt

```diff
@@ -106,6 +106,9 @@ option(BUILD_PORTAUDIO "Build with support for PortAudio audio backend." ON)
 option(BUILD_PORTMIDI "Build with support for PortAudio's MIDI features." ${BUILD_PORTAUDIO}) # PortAudio required
 option(BUILD_PCH "Build using precompiled headers." ON)
 option(BUILD_FOR_WINSTORE "Build for the Windows Store." OFF)
+option(COVERAGE "Build with instrumentation to record code coverage." OFF)
+
+set (ENABLE_COVERAGE (COVERAGE AND CMAKE_BUILD_TYPE MATCHES "DEBUG"))
```

Comment:
This is not add insensitive, or is it? There was a change recently so we don't fly ti to be in all caps anymore

Reply:
The check for DEBUG is case-sensitive, but so is the existing check for DEBUG later in the file that I modelled this on...

Reply:
Well but only right before that it gets upshifted, see https://github.com/musescore/MuseScore/blob/6fbf7279eb99bb815c1fb66f00292ea3888e38c6/CMakeLists.txt#L563

Reply:
Ah awesome, thanks